tsconfig.json

```diff
@@ -1,10 +1,3 @@
 {
-  // https://nuxt.com/docs/guide/concepts/typescript
```

Comment:
Don't know about this comment, is this json5? My editor doesn't like this atleast since comments in JSON don't exist.